catalog/dags/common/constants.py

```diff
@@ -29,11 +29,9 @@
 }
 XCOM_PULL_TEMPLATE = "{{{{ ti.xcom_pull(task_ids='{}', key='{}') }}}}"
 
-POSTGRES_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
-OPENLEDGER_API_CONN_ID = os.getenv("OPENLEDGER_API_CONN_ID", "postgres_openledger_api")
-POSTGRES_API_STAGING_CONN_ID = os.getenv(
-    "POSTGRES_API_STAGING_CONN_ID", "postgres_openledger_api_staging"
-)
-AWS_CONN_ID = os.getenv("AWS_CONN_ID", "aws_conn_id")
+POSTGRES_CONN_ID = "postgres_openledger_upstream"
```

Comment:
Should we add a follow-up issue to standardize and document these constants?

Reply:
An issue for documenting them would be good! Standardizing them would be a bit harder for _existing_ ones because it would require some extra work in production so we don't break things when the code is deployed. I can make an issue for it and make note of that there 🙂